LICENSE

```diff
@@ -0,0 +1,66 @@
+MIT License
+
+Copyright (c) 2020 Africa's Voices Foundation (AVF)
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in all
+copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+
+
+3RD PARTY EXCEPTIONS
+
+The geometry data for Somalia is a redistribution of a dataset from OCHA Somalia,
+with features and properties modified by Africa's Voices Foundation.
+Those files, along with their source and licence, are:
+
+- `geojson/somalia_regions.geojson`: Adapted from data produced by OCHA Somalia.
+Retrieved from https://data.humdata.org/dataset/somalia-administrative-boundaries
+January 2020. Licence "Other: humanitarian use only"
+
+- `geojson/somalia_districts.geojson`: Adapted from data produced by OCHA Somalia.
+Retrieved from https://data.humdata.org/dataset/somalia-administrative-boundaries
+April 2020. Licence "Other: humanitarian use only"
+
+Terms and Conditions of Use
```

Comment:
I don't _think_ this licence applies to the Somali datasets, but correct me if you think otherwise

Reply:
Yes it does, I downloaded them from https://data.humdata.org/dataset/somalia-administrative-boundaries.